app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -58,6 +65,26 @@ def get_vms
     process_collection(@inv[:vms], :vms) { |vm| parse_vm(vm) }
   end
 
+  def get_vapp_templates
+    @inv[:orgs].each do |org|
+      org.catalogs.each do |catalog|
+        next if catalog.is_published && !@options.get_public_images || !catalog.is_published && !@options.get_private_images
```

Comment:
I know amazon does it this way but I don't know what the use case would be for turning off collecting private images unless it is common to have thousands of private catalogs that you don't care about.  Amazon is the only provider that I see that allows you to turn off private image collection.